src/lib/theme/default.ts

```diff
@@ -240,7 +240,7 @@ export default {
     },
   },
   carousel: {
-    base: 'relative h-full w-full min-h-56 sm:min-h-64 xl:min-h-80 2xl:min-h-96',
+    base: 'relative h-full w-full h-56 sm:h-64 xl:h-80 2xl:h-96',
```

Comment:
I think that the we should replace the `h-full` and `w-full` with `h-min-full` and `w-min-full`, but we need to test. What do you think @tulup-conner?

Reply:
I think that could be fine as well.